price-estimator/src/models.rs

```diff
@@ -132,4 +146,53 @@ mod tests {
         });
         assert_eq!(json, expected);
     }
+
+    #[test]
+    fn sanitize_orders_works() {
```

Comment:
nit: I find testing that it "works" a bit lax (requires implicit knowledge of how this method is supposed to work or jumping to the doc string of the method which can be outdated). Can we call it `sanitize_orders_sums_volume_at_same_price`